enterprise/causal-clustering/src/main/java/org/neo4j/causalclustering/core/CausalClusteringSettings.java

```diff
@@ -101,10 +102,21 @@
     public static final Setting<Integer> raft_in_queue_max_batch =
             setting( "causal_clustering.raft_in_queue_max_batch", INTEGER, "64" );
 
-    @Description( "Expected number of Core machines in the cluster" )
+    @Description( "Expected number of Core machines in the cluster before startup" )
```

Comment:
I think we can do a better job documenting the behaviour here. The text here will show up in the reference manual. I am specifically missing a short description, which captures the gist of it. Longer descriptions go in the manual. This can be used as inspiration:

https://stackoverflow.com/questions/43103105/monitoring-a-causal-clusters-redundancy

Comment applies to all three settings.

Reply:
Done.